catalogs/catalog-hive/bin/generated-sources/annotations/org/apache/gravitino/catalog/hive/ImmutableConfElement.java

```diff
@@ -0,0 +1,225 @@
+  }
+
+  /**
+   * Builds instances of type {@link ImmutableConfElement ImmutableConfElement}.
+   * Initialize attributes and then invoke the {@link #build()} method to create an
+   * immutable instance.
+   * <p><em>{@code Builder} is not thread-safe and generally should not be stored in a field or collection,
+   * but instead used immediately to create instances.</em>
+   */
+  @Generated(from = "CachedClientPool.ConfElement", generator = "Immutables")
+  @NotThreadSafe
+  public static final class Builder {
+    private static final long INIT_BIT_KEY = 0x1L;
+    private long initBits = 0x1L;
+
+    private @Nullable String key;
+    private @Nullable String value;
+
+    private Builder() {
+    }
+
+    /**
+     * Fill a builder with attribute values from the provided {@code ConfElement} instance.
+     * Regular attribute values will be replaced with those from the given instance.
+     * Absent optional values will not replace present values.
+     * @param instance The instance from which to copy values
+     * @return {@code this} builder for use in a chained invocation
+     */
+    @CanIgnoreReturnValue 
+    public final Builder from(CachedClientPool.ConfElement instance) {
+      Objects.requireNonNull(instance, "instance");
+      key(instance.key());
+      @Nullable String valueValue = instance.value();
+      if (valueValue != null) {
+        value(valueValue);
+      }
+      return this;
+    }
+
+    /**
+     * Initializes the value for the {@link CachedClientPool.ConfElement#key() key} attribute.
+     * @param key The value for key 
+     * @return {@code this} builder for use in a chained invocation
+     */
+    @CanIgnoreReturnValue 
+    public final Builder key(String key) {
+      this.key = Objects.requireNonNull(key, "key");
+      initBits &= ~INIT_BIT_KEY;
+      return this;
+    }
+
+    /**
+     * Initializes the value for the {@link CachedClientPool.ConfElement#value() value} attribute.
+     * @param value The value for value (can be {@code null})
+     * @return {@code this} builder for use in a chained invocation
+     */
+    @CanIgnoreReturnValue 
+    public final Builder value(@Nullable String value) {
+      this.value = value;
+      return this;
+    }
+
+    /**
+     * Builds a new {@link ImmutableConfElement ImmutableConfElement}.
+     * @return An immutable instance of ConfElement
+     * @throws java.lang.IllegalStateException if any required attributes are missing
+     */
+    public ImmutableConfElement build() {
+      if (initBits != 0) {
+        throw new IllegalStateException(formatRequiredAttributesMessage());
+      }
+      return new ImmutableConfElement(key, value);
+    }
+
+    private String formatRequiredAttributesMessage() {
+      List<String> attributes = new ArrayList<>();
+      if ((initBits & INIT_BIT_KEY) != 0) attributes.add("key");
+      return "Cannot build ConfElement, some of required attributes are not set " + attributes;
+    }
+  }
+}
```

Comment:
Hi @Jash2606 is this file you wanted or you added mistakenly?

Reply:
@jerryshao   I got this file while running up the prettier , So Should i keep it or remove it ?

Reply:
I don't know who generates this file, from my point it is not necessary.